client/test/e2e/employees/employees.spec.js

```diff
@@ -0,0 +1,175 @@
+/* jshint expr:true */
+/* global element, by, beforeEach, inject, browser */
+
+var chai = require('chai');
+var expect = chai.expect;
+
+var FU = require('../shared/FormUtils');
+var helpers = require('../shared/helpers');
+var components = require('../shared/components');
+
+helpers.configure(chai);
+
+describe('Employees Module', function () {
+  'use strict';
+
+  var path = '#/employees';
+
+  var employee = {
+      code : 'HBB80',
+      prenom : 'Sherlock',
+      name : 'Holms',
+      postnom : 'Doyle',
+      sexe : 'M',
+      dob : '06/30/1960',
+      date_embauche : '05/17/1997',
+      nb_spouse : 1,
+      nb_enfant : 2,
+      bank : 'BIAC',
+      bank_account : '00-99-88-77',
+      email : 'me@info.com',
+      adresse : '221B Baker Street'
+  };
+
+  var defaultEmployee = 2;
+  var employeeRank = helpers.random(defaultEmployee);
+
+  /** locations to be used in the patient select */
+  var locations = [
```

Comment:
Just leaving a note here - we should remember to move these locations into a helper function.